.github/ISSUE_TEMPLATE/new-content.md

```diff
@@ -4,7 +4,8 @@ about: Submit content to be reviewed and published
 title: Content's title
 labels: content
 assignees: ''
-
+twitter_body: Specific body for twitter
+twitter_hashtags: ['list', 'of', 'hashtags']
```

Comment:
In which way do propose to use this list of hashtags? They will be append to the of tweet? or do you see other use case?

Reply:
They will be happened to the tweet... with defaults ones like "#sre" ...